config/application.yml.default

```diff
@@ -219,7 +219,7 @@ pinpoint_voice_pool_size: 5
 piv_cac_service_url: https://localhost:8443/
 piv_cac_service_timeout: 5.0
 piv_cac_verify_token_url: https://localhost:8443/
-platform_authentication_enabled: true
+platform_authentication_enabled: false
```

Comment:
If we're repurposing the feature flag to control new sign-ups, should we consider renaming it as well?

e.g.

```suggestion
platform_authentication_new_setups_enabled: false
```